### PR TITLE
Fix numpy run requirement

### PR DESCRIPTION
**Pull request by rmax**
Conda-smithy fails to generate the build scripts with older numpy versions.

### CODE REVIEW

**Comment by rmax** [comment]
This fixes the builds from #4

File: recipe/meta.yaml [hunk 1]
```diff
@@ -35,7 +35,7 @@ requirements:
     - pystan >=2.14
     # Pystan models depend on numpy and as they are compiled and pickled we
     # need to pin numpy version.
-    - numpy 1.10.*    
+    - numpy 1.11.*
 
   run:
     - python
```
Comment (isuruf) [inline recipe/meta.yaml]: Can you add `numpy >=1.11` here? Also changing numpy to `1.10` in both build and run works for me. (When numpy is not in run for 1.10, it fails like you mentioned)

Comment (rmax) [reply recipe/meta.yaml]: I think the idea is to use the minimum numpy version at build time so the pickled objects have forward compatibility. When using `1.10` the problem is that `conda-smithy rerender` removes all CI files. See https://github.com/conda-forge/conda-smithy/issues/601

Comment (isuruf) [reply recipe/meta.yaml]: It works correctly if you have
```
   build:
      - numpy 1.10.*
   run:
      - numpy >=1.10
```

You only had
```
   build:
      - numpy 1.10.*
```

Comment (rmax) [reply recipe/meta.yaml]: @isuruf you are right. Do you mind to chime in https://github.com/conda-forge/conda-smithy/issues/601 ?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -34,14 +34,16 @@ requirements:
...
+    # need to pin numpy version. Note that 1.10 is not available for win+py36.
+    - numpy 1.10.*  # [not win or py<=35]
+    - numpy 1.11.*  # [win and py>=36]
 
   run:
     - python
     - matplotlib
     - pandas >=0.18.1
     - pystan >=2.14
+    - numpy >=1.10
```
Comment (isuruf) [inline recipe/meta.yaml]: You need to change this too.

**Pull request merged**